pytrip/vdx.py

```diff
@@ -1210,6 +1210,30 @@ def get_min_max(self):
         self.temp_max = temp_max
         return temp_min, temp_max
 
+    def is_fully_contained(self):
+        try:
+            [min_pos_x, min_pos_y, min_pos_z], [max_pos_x, max_pos_y, max_pos_z] = self.get_min_max()
+        except TypeError:
+            # get_min_max can return NoneTypes if a VOI is located outside of the patient
+            return False
+
+        if [min_pos_x, min_pos_y, min_pos_z] == [max_pos_x, max_pos_y, max_pos_z]:
+            # get_min_max can return equal min and max values if a VOI is partially contained in the patient
+            return False
+
+        return self.is_x_contained(min_pos_x, max_pos_x) and \
+               self.is_y_contained(min_pos_y, max_pos_y) and \
+               self.is_z_contained(min_pos_z, max_pos_z)
+
+    def is_x_contained(self, min_pos, max_pos):
```

Comment:
These methods looks like something that shoun't be exposed in class API, consider making them "private".
In order to do so, rename them to `_is_x_contained`